src/Projections/ProjectionStatus.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+/**
+ * This file is part of prooph/event-store.
+ * (c) 2014-2020 Alexander Miertsch <kontakt@codeliner.ws>
+ * (c) 2015-2020 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Prooph\EventStore\Projections;
+
+final class ProjectionStatus
```

Comment:
Use the existing ProjectionDetails class. No need to duplicate it.

Reply:
@enumag right, didn't notice that